### DISCUSSION TITLE
Hide ref redering for Tex

### DISCUSSION THREAD

**Discussion by vakker** [LazyVim/LazyVim#4301] [Q&A]
I'm using the Tex plugin (`lazyvim.plugins.extras.lang.tex`), I set `vim.opt.conceallevel = 0`, but I can't find the option to remove the ref rendering as shown on the image:
![image](https://github.com/user-attachments/assets/89daf668-08fb-4832-9df2-2aa0ab892107)

How do I get rid of that?
Thanks!

**Comment by LucasKook** [comment]
One can toggle this behavior with `leader` + `u` + `h` (Enable Inlay Hints). The leader key is the space key by default.
